src/contexts/OnChainValuesContextProvider.tsx

```diff
@@ -0,0 +1,44 @@
+// Copyright 2021 Parity Technologies (UK) Ltd.
```

Comment:
This context can include all the different API calls

Reply:
also i'm opened to listen suggestions for a better context provider name ... :)

Reply:
forget about these two comments. 

1) i think calling this context Subscriptions is quite descriptive.
2) I changed my mind regarding this context usage ... i think it should be only charge of subscription so i just force the re render of components only when the subscriptions are updated.

Reply:
> The commented code could be easily found if you self-reviewed the PR first. I don't want to block any further work though, so please proceed.

This issue was solved on PR #181  which overwrite the outdated testing code for `useBlockInfo.test.ts` as i basically had to rewrote it because it has now a complete different behavior.